mrp_partial_production/models/mrp.py

```diff
@@ -44,10 +20,10 @@ def _compute_qty_to_produce(self):
             record.qty_available_to_produce = total
 
     qty_available_to_produce = fields.Float(
-        string='Quantity Available to Produce',
+        string=_('Quantity Available to Produce'),
```

Comment:
1. Why do you think that you need _()?
2. If the string if the same of the name fields is not necessary

Reply:
ready change